### PR TITLE
Add suffixes to typos from "coo" to "coy"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -14438,10 +14476,18 @@ copmutations->computations
 copntroller->controller
 coponent->component
 coponents->components
+coporate->corporate
+coporation->corporation
+coporations->corporations
 copoying->copying
 coppermines->coppermine
 coppied->copied
+coppies->copies
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:14485]: Perhaps `coppices`, and `coppiced` in the line before?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -14815,8 +14877,14 @@ corresspondingly->correspondingly
 corressponds->corresponds
 corret->correct
 correted->corrected
+correting->correcting
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:14880]: I guess occurrences of `correlating` would be rare at best, wouldn't they?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -14913,6 +14984,13 @@ corupts->corrupts
 corus->chorus
 corvering->covering
 cosed->closed
+cosist->consist
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:14987]:
```suggestion
cosist->consist, cosiest,
```

File: codespell_lib/data/dictionary_rare.txt [hunk 4]
```diff
@@ -55,7 +55,10 @@ coo->coup
 copping->coping, copying, cropping,
 corespondent->correspondent
 corespondents->correspondents
-covert->convert
+costumer->customer
+costumers->customers
+covert->convert, cover, covet,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary_rare.txt:60]: Not sure about `covert`, even in the rare dictionary, but I guess typos of `convert` might be more common than `covert` itself in the context of code.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -14942,8 +15026,38 @@ costructions->constructions
...
+cosumables->consumables
+cosume->consume, costume,
+cosumed->consumed
+cosumer->consumer
+cosumers->consumers
+cosumes->consumes, costumes,
+cosuming->consuming
+cosumption->consumption
+cosumptions->consumptions
+cotact->contact
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:15057]: No `coact` typos?

Comment (int-y1) [reply codespell_lib/data/dictionary.txt:15057]: I couldn't find any typos. `coact` is too rare.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:14880]: This occurs a couple of times. I will add this, along with other suffixes.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:14485]: This is rare, but exists. I'll add these.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -14339,7 +14339,38 @@ coolent->coolant
 coolot->culotte
 coolots->culottes
 coomand->command
+coomanded->commanded
+coomanding->commanding
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:14343]: These are too rare.

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -14339,7 +14339,38 @@ coolent->coolant
...
+coomands->commands
+cooment->comment
+coomented->commented
+coomenting->commenting
+cooments->comments
+cooming->coming
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:14349]: I will add `cooling`. I couldn't find examples of the other typos.

File: codespell_lib/data/dictionary_rare.txt (hunk 4 above)
Comment (int-y1) [reply codespell_lib/data/dictionary_rare.txt:60]: Is there anything to do here?

File: codespell_lib/data/dictionary.txt (hunk 6 above)
[the remark this answers is not in the document]
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:14343]: Perhaps `commander` then, but let's leave it for future PRs.

File: codespell_lib/data/dictionary_rare.txt (hunk 4 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary_rare.txt:60]: I was not certain about `covert` appearing at all, even in the **rare** dictionary.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**